src/inference_profiler.cc

```diff
@@ -1413,8 +1413,12 @@ InferenceProfiler::CollectData(
     PerfStatus& summary, uint64_t window_start_ns, uint64_t window_end_ns,
     std::vector<RequestRecord>&& request_records)
 {
-  ProfileDataCollector::InferenceLoadMode id{
-      summary.concurrency, summary.request_rate};
+  ProfileDataCollector::InferenceLoadMode id{};
+  if (dynamic_cast<CustomRequestScheduleManager*>(manager_.get())) {
+    id = {0, 0.0};
+  } else {
+    id = {summary.concurrency, summary.request_rate};
+  }
```

Comment:
PA's profile export JSON currently requires a concurrency/request rate associated with each export--this is an artifact of an old design and should be updated eventually.

 Fixed schedule mode has no concept of concurrency or request rate, but since a value is needed to create the profile export JSON, this code specifies the export's "concurrency/request rate" as zero, a sentinel value.

Code on GAP side (see [here](https://github.com/triton-inference-server/perf_analyzer/pull/293/files#diff-9d974f7d793b676e0d1155531fdc4d57c42ad7b1e02f8e4123285d5cfe041205R93)) will be expecting a profile export with concurrency/request rate of zero for the fixed schedule case.